File: ARTHUR TENORIO/FACTORY_METHOD/ERRADO/NOTIFIER/EmailNotification.js
const Notification = require("./Notification");

class EmailNotification extends Notification {
  send(recipient, message) {
    console.log(`[Email] Enviando email para ${recipient}`);
    console.log(`Conteúdo: ${message}`);
    console.log("Email enviado com sucesso!");
  }
}

module.exports = EmailNotification;

File: RAFAEL_ROANI_GONCALVES/src/decorator/without/EmailNotification.java
package decorator.without;

public class EmailNotification extends Notification {

    private String emailAddress;
    
    public EmailNotification(String message, String emailAddress) {
        super(message);
        this.emailAddress = emailAddress;
    }
    
    @Override
    public void send() {
        super.send();
        System.out.println("Also sending via Email to: " + emailAddress);
    }

}
